libs/response-extensions.js

```diff
@@ -62,9 +64,11 @@ module.exports.send = (options, req, res) => {
       }
 
       if (data) {
-        if (data instanceof Buffer) {
+        if (typeof data === 'string') {
+          contentType = contentType || TYPE_PLAIN
```

Comment:
with this we are losing performance, I put in separate functions because directly towards its objective, reassigning parameters is not a good practice.

Reply:
Hi Jesus, we are not loosing performance with this, we are gaining 2-4k more requests per second with this updates. I agree, this is not probable nice looking or good practices, but this is very performant, at least according to my stress tests.

Reply:
@jkyberneees We should write those tests too, so we can have better quality code.